components/cluster-secret-store/internal-production/kustomization.yaml

```diff
@@ -0,0 +1,17 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - ../base
+patches:
+  - path: approle-id-patch.yaml
+    target:
+      name: appsre-stonesoup-vault
+      kind: ClusterSecretStore
+      group: external-secrets.io
+      version: v1beta1
+  - path: approle-id-patch.yaml
```

Comment:
patch for appsre-vault is not needed since we do not need appsre-vault ClusterSecretStore